.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Description
+Please provide a brief description of the changes in this PR.
+
+## Type of Change
+Please prefix your PR title with one of the following:
+- [ ] `[Bugfix]` - Bug fixes
```

Comment:
Could we refine this section?

I wonder if this is too long a list for developers. Lmk what you think